kbpagesd/main.go

```diff
@@ -23,7 +23,7 @@ var (
 
 func init() {
 	flag.BoolVar(&fProd, "prod", false, "disable development mode")
```

Comment:
Should this be true now by default?

Reply:
I'd prefer having to explicitly set it. Same concept as requiring to set `KEYBASE_RUN_MODE` for non-dev environments. WDYT?

Reply:
Sounds good.